### ISSUE TITLE
Pretrained Inception V4 to Keras Application Folder

### ISSUE THREAD

**Issue by amangalampalli** [tensorflow/tensorflow#33749]
Would it be possible to add Inception V4 to the Keras Applications folder?

System/User Specifics: Tensorflow 2.0 with `tf.keras` module

Due to recent discoveries made in neural networks and also according to many benchmarking standards, Inception V4 is one of the best networks available with very high accuracies in both Top 5 and Top 1. 

Inception V4 should be included because it has been consistently proven to be one of the most accurate networks for Image Recognition. It will benefit many users who want a network with very high accuracy and also have the resources to run it.

**Comment by VyBui** [comment]
It has been 3 months, do we have any info about this request?

**Comment by tanzhenyu** [comment]
keras applications is actually in the merging process with tf model garden. Maybe we should have a formal discussion of where this should go.

**Comment by tanzhenyu** [comment]
I think we should make this contributions welcome. @Alpheron Would you be interested?

**Comment by Alpheron** [comment]
So I think tensorflow-slim has an Inception V4 model which has been written which can be found [here](https://github.com/tensorflow/models/blob/master/research/slim/nets/inception_v4.py). However, according to the documentation saying that Slim is not supported with Tensorflow 2.0, I think it would be fair to say that this model must be converted sometime in the future to be able to work with Keras. In fairness, the models found [here](https://github.com/tensorflow/models/blob/master/research/slim/README.md#pre-trained-models are all very capable models not all of which are supported. I am not 100% sure about if the Tensorflow Model Garden was also merged recently. In that case, it may not make sense to only implement Inception V4 to Keras Applications, but it may make sense to port the entire Tensorflow Slim model library to Keras Applications. That might sound a bit ambitious but I think the models all are very capable in different use cases and may be of use to people. As far as Inception V4 goes, I think that I will be able to port that to Keras and will be able to issue a PR sometime in the very near future (around a week or so).

**Comment by mohantym** [comment]
Hi @amangalampalli !

Could you consider this duplicate  to #[1153](https://github.com/keras-team/keras-cv/issues/1153) and put your thoughts in that issue.

Thank you!

**Comment by amangalampalli** [comment]
Yeah, sorry for the long delays, I will close this issue and will speak on the other issue.
